### DISCUSSION TITLE
DIIP V2 specifications

### DISCUSSION THREAD

**Discussion by k00ij** [FIDEScommunity/DIIP#12] [General]
- OpenID for Verifiable Credential Issuance [WG draft 12](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html)
- OpenID for Verifiable Presentations [WG draft 20](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html)
- SIOP V2 [WG draft 13](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html)
- DIF Presentation Exchange [V2.0](https://identity.foundation/presentation-exchange/spec/v2.0.0/)
- W3C Verifiable Credential JWT 
- DID methods: DID:WEB and DID:JWK 
- Signature types ES256 
- Revocation method: StatusList 2021

**Comment by JelleMillenaar** [comment]
StatusList2021 has been renamed to be [Bitstring Status List v1.0](https://www.w3.org/TR/vc-bitstring-status-list/). Unfortunately, in this update, they seem to also adopt Verifiable Credentials v2.0 terminology such as ValidFrom and ValidUntil. This might make it difficult to apply this standard in combination with VC v1.1. Perhaps we need to consider an older version of StatusList2021. I believe [this](https://www.w3.org/TR/2023/WD-vc-status-list-20230427/) is the last stable draft that might still work with VC v1.1, but I am not an expert.

**Reply by nklomp** [reply]
Yes we are sticking with StatusList2021 for V2. Bitstring but also IETF OAuth Status Lists and potentially others will be part of V3

**Comment by nklomp** [comment]
DIIP v2 has just been deployed: https://dutchblockchaincoalition.github.io/DIIP/
